elements/pfe-navigation/README.md

```diff
@@ -0,0 +1,65 @@
+# PatternFly Element | Navigation Element
+
+## Navigation element
+
+This element will give you a top-level navigation element (not for use as subnavigation).
+
+### Slots
+
+- `skip`: This slot contains all the content for the skip navigation. Best practice is to use an unordered list with link tags that link to anchors on the page relevant to the users. These anchors can include things like: "main content", "registration form", "contact us", etc. Currently this is only available to screen-readers.
+- `logo`: This slot should contain the site's main logo.
+- `search`: This slot, when assigned to a navigation-item component, renders a dropdown in the navigation for the search functionality. Inside the tray for the slot, we recommend tagging the search form or search functionality that includes the input and submit button with the attribute `pfe-navigation--mobile-search`. The element inside the search slot that is tagged with this attribute will be copied into the mobile menu and appear above the accordion for the main navigation.
+- `language`: This slot, when assigned to a navigation-item component, renders the dropdown for the user to select the site language.
+- `mobile-language`: A simple link to a landing page containing language switching information.  Example: `<a href="/url-to-language-page" slot="mobile-language" pfe-icon="user" hidden>English</a>`.
+- `login`: This slot, when assigned to a navigation-item component, renders the dropdown for the user to log into the site.
+- `mobile-login`: A simple link to a landing page containing language switching information.  Example: `<a href="/login" slot="mobile-login" pfe-icon="user" hidden>Login/Register</a>`.
```

Comment:
Maybe this should be: 
```
mobile-login: A simple link to a page containing a login form. Example:...
```

Reply:
Same question about the `hidden` attribute here.

Reply:
+1 updating now